src/components/common/SelectDropdown.jsx

```diff
@@ -0,0 +1,38 @@
+import Select from 'react-tailwindcss-select';
```

Comment:
Just interesting why you installed a new library instead of using[ Select from MUI ](https://mui.com/material-ui/react-select/). No right answer, just wandering.

Reply:
This option could be styled using the already configured tailwind styles.